docs/docs/reference/contextual/extension-methods.md

```diff
@@ -92,13 +92,13 @@ By contrast, type arguments matching type parameters following `extension` can b
 only if the method is referenced as a regular method:
 
 ```scala
-List[String]("a", "bb", "ccc").sumBy(_.length)
+sumBy[String](List("a", "bb", "ccc"))(_.length)
```

Comment:
@odersky it seems the original doc was referencing the `sumBy` method still as an extension method (as spotted by @julienrf). Can you confirm this change is correct?

Reply:
Yes, correct